modules/opencv/metadata.json

```
@@ -0,0 +1,18 @@
+{
+    "homepage": "https://opencv.org/",
+    "maintainers": [
+        {
+            "email": "ryangyang99@hotmail.com",
+            "github": "Ryang20718",
+            "name": "Ryan Yang",
+            "github_user_id": 31294356
+        }
+    ],
+    "repository": [
+        "https://github.com/opencv/opencv/archive/refs/tags/4.11.0.tar.gz"
```

Should only be the repo
```suggestion
        "https://github.com/opencv/opencv"
```